### PR TITLE
Re-organize the content of the `*.proto` files.

### PR DESCRIPTION
**Pull request by nalepae**
**What type of PR is this?**
Other

**What does this PR do? Why is it needed?**
The PR re-organizes the content of the `*.proto` files.

The content of the `*.proto` files is sorted by hard fork, then with a top-down fashion.

Sorting first by hard fork lets the reader to easily see new or modified fields. Then, sorting with a top-down fashion lets the user to first see the big picture, then to dive into details.

Also, the `new in <hard fork>` mentions are only written for the given hard fork. Thus, it'll avoid in the future the majority of the fields, not initially present in phase 0, to have the `new in <hard fork> mention`.

This commit does not bring any new functional change.

**Other notes for review**
If the proposed organisation does not suit you, I have no objection at all to change it.
My main concern is to have something consistent in the same file and across the files.

The diff is quite large, but is actually mostly due to the re-organization of generated files due to the re-organization of the 
`*.proto` files.

**Acknowledgements**

- [x] I have read [CONTRIBUTING.md](https://github.com/prysmaticlabs/prysm/blob/develop/CONTRIBUTING.md).
- [x] I have made an appropriate entry to [CHANGELOG.md](https://github.com/prysmaticlabs/prysm/blob/develop/CHANGELOG.md).
- [x] I have added a description to this PR with sufficient context for reviewers to understand this PR.

### CODE REVIEW

**Review by terencechain — APPROVED** [review]

File: proto/prysm/v1alpha1/attestation.proto [hunk 1]
```diff
@@ -24,36 +24,16 @@ option java_outer_classname = "AttestationProto";
 option java_package = "org.ethereum.eth.v1alpha1";
 option php_namespace = "Ethereum\\Eth\\v1alpha1";
 
-message Attestation {
-    // A bitfield representation of validator indices that have voted exactly
-    // the same vote and have been aggregated into this attestation.
-    bytes aggregation_bits = 1 [(ethereum.eth.ext.ssz_max) = "2048", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitlist"];
+// ----------------------------------------------------------------------------
+// Phase0, Bellatrix, Capella & Deneb
```
Comment (rkapka) [inline proto/prysm/v1alpha1/attestation.proto:28]:
```suggestion
// Phase0, Altair, Bellatrix, Capella & Deneb
```

File: proto/prysm/v1alpha1/attestation.proto [hunk 2]
```diff
@@ -67,31 +47,22 @@ message AggregateAttestationAndProof {
...
+    bytes signature = 3 [(ethereum.eth.ext.ssz_size) = "96"];
 }
 
-message SignedAggregateAttestationAndProofElectra {
-    // The aggregated attestation and selection proof itself.
-    AggregateAttestationAndProofElectra message = 1;
-
-    // 96 byte BLS aggregate signature signed by the aggregator over the message.
-    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+message SingleAttestation {
```
Comment (rkapka) [inline proto/prysm/v1alpha1/attestation.proto:61]: This is an Electra type

File: proto/prysm/v1alpha1/beacon_state.proto [hunk 3]
```diff
@@ -175,6 +132,71 @@ message DepositMessage {
...
+//    block_hash: Hash32
+//    parent_hash: Hash32
+//    total_difficulty: uint256
+message PowBlock {
+  bytes block_hash = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+  bytes parent_hash = 2 [(ethereum.eth.ext.ssz_size) = "32"];
+  bytes total_difficulty = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+}
+
+
```
Comment (rkapka) [inline proto/prysm/v1alpha1/beacon_state.proto:147]:
```suggestion
```

File: proto/prysm/v1alpha1/attestation.proto (hunk 2 above)
Comment (nalepae) [reply proto/prysm/v1alpha1/attestation.proto]: Fixed in 44b89cd097b9ac5e5ef016c6e79d2eaff4bafed9.

**Review by rkapka — APPROVED** [review]

**Pull request merged**